lib/seafoam/cfg/disassembler.rb

```diff
@@ -16,7 +16,14 @@ def disassemble(nmethod, print_comments)
         comments = nmethod.comments
         comments_n = 0
 
-        cs = Crabstone::Disassembler.new(Crabstone::ARCH_X86, Crabstone::MODE_64)
+        case [nmethod.code.arch, nmethod.code.arch_width]
+        when %w[AMD64 64]
+          crabstone_arch = [Crabstone::ARCH_X86, Crabstone::MODE_64]
+        else
+          raise "Unsupported architecture #{nmethod.code.arch} and bit width #{nmethod.code.arch_width}"
```

Comment:
Maybe put something like: "Currently unsupported architecture #{nmethod.code.arch} and bit width #{nmethod.code.arch_width}. Please open a github issue at Shopify/seafoam to see if support can be quickly added."

So we can get other people to help us complete the mapping if they use it.

Reply:
Thanks I switched to 'unknown' since Crabstone likely does support it, we just don't know what it looks like in the CFG files yet.